### DISCUSSION TITLE
Qual a ideologia/princípio você mais se identifica/gosta no framework/library que você?

### DISCUSSION THREAD

**Discussion by AbraaoAlves** [frontendbr/forum#424] [Perguntas e Respostas]
Uma pergunta especifica surgiu enquanto escrevia uma resposta na issue #404. 

Acredito que podemos comparar tecnologias sem precisar falar mal delas, então vamos lá! Quanto mais pontos de vistas, mais teremos informações para escolher a tecnologia do próximo projeto.

**Comment by felquis** [comment]
Interoperabilidade, posso deixar paradinho em produção, e continuará funcionando daqui 5, 10 anos.
De forma que no futuro se eu precisar alterar algo não vou precisar fazer upgrades de dependências, mudar APIs e atualizar tudo para finalmente poder fazer o que eu gostaria de fazer.

Eu realmente amo ficar meses sem olhar um projeto, e quando foco nele novamente tudo está funcionando exatamente da forma com que eu deixei.
